core/modules/widgets/navigator.js

```diff
@@ -44,7 +44,8 @@ NavigatorWidget.prototype.render = function(parent,nextSibling) {
 		{type: "tm-fold-tiddler", handler: "handleFoldTiddlerEvent"},
 		{type: "tm-fold-other-tiddlers", handler: "handleFoldOtherTiddlersEvent"},
 		{type: "tm-fold-all-tiddlers", handler: "handleFoldAllTiddlersEvent"},
-		{type: "tm-unfold-all-tiddlers", handler: "handleUnfoldAllTiddlersEvent"}
+		{type: "tm-unfold-all-tiddlers", handler: "handleUnfoldAllTiddlersEvent"},
+		{type: "tm-manage-acl", handler: "handleManageACLTiddlersEvent"}
```

Comment:
The navigator widget is part of the TiddlyWiki core. It cannot include components that are specific to MWS.

In this case, rather than using a message handler, the most appropriate technique would be for the MWS client plugin to provide a global function that yields the URL of the appropriate admin page.